lib/phoenix_live_view.ex

```diff
@@ -1003,6 +1003,14 @@ defmodule Phoenix.LiveView do
     Phoenix.LiveView.Channel.send_update(module, id, assigns)
   end
 
+  def send_update_after(module, time_in_milliseconds, assigns) do
```

Comment:
@josevalim Please, take a look if I'm in the right direction. I got my application working with this change.

If that sounds right, I'll proceed with tests & docs :)

Reply:
To be honest, I think that is it. Let me know if something is missing 🙌

Reply:
This looks great! My only question is if the signature should be `send_update_after(module, time_in_milliseconds, assigns)` or `send_update_after(module, assigns, time_in_milliseconds)`. `Process.send_after` puts the time as the last argument, which we could do here, but it means you need to wrap the keyword list in brackets. I'd think though being consistent with Elixir could be nice.

@feliperenan wdyt? @chrismccord, any preferences?

Reply:
Well, good point. I choose this way in order to avoid brackets but I do think consistency with Elixir is also important. I've push this change then you can take a look how it looks like :).